### ISSUE TITLE
finally

### ISSUE THREAD

**Issue by AntiWarforever** [Mffff4/cityholderbot#1]
thanks very ... but 
```
from session not created: This version of ChromeDriver only supports Chrome version 129
Current browser version is 125.0.6422.112
Stacktrace:
        GetHandleVerifier [0x00D55523+24195]
        (No symbol) [0x00CEAA04]
        (No symbol) [0x00BE2093]
        (No symbol) [0x00C18C5F]
        (No symbol) [0x00C17D09]
```

**Comment by Mffff4** [comment]
> thanks very ... but 
> 
> ```
> 
> from session not created: This version of ChromeDriver only supports Chrome version 129
> 
> Current browser version is 125.0.6422.112
> 
> Stacktrace:
> 
>         GetHandleVerifier [0x00D55523+24195]
> 
>         (No symbol) [0x00CEAA04]
> 
>         (No symbol) [0x00BE2093]
> 
>         (No symbol) [0x00C18C5F]
> 
>         (No symbol) [0x00C17D09]
> 
> ```

Hi. And what kind of system do you use? If you use ubuntu, then write the commands:
``` 
sudo apt update
sudo apt install -y chromium-chromedriver
```

Or use docker, everything should start without problems in it.

**Comment by AntiWarforever** [comment]
i windows
```
te-packages (from -r requirements.txt (line 34)) (5.2)
ERROR: Could not find a version that satisfies the requirement chromium-chromedriver (from versions: none)

[notice] A new release of pip is available: 24.2 -> 24.3.1
[notice] To update, run: python.exe -m pip install --upgrade pip
ERROR: No matching distribution found for chromium-chromedriver
Skipping .env copying
Press any key to continue . . .
```

**Comment by Mffff4** [comment]
Wait for it. I'm rewriting the code for another library.

**Comment by Mffff4** [comment]
Hi again, I released an update, you can check it out.

> i windows
> 
> ```
> te-packages (from -r requirements.txt (line 34)) (5.2)
> ERROR: Could not find a version that satisfies the requirement chromium-chromedriver (from versions: none)
> 
> [notice] A new release of pip is available: 24.2 -> 24.3.1
> [notice] To update, run: python.exe -m pip install --upgrade pip
> ERROR: No matching distribution found for chromium-chromedriver
> Skipping .env copying
> Press any key to continue . . .
> ```

**Comment by Mffff4** [comment]
FIXED
